src/dpi.erl

```diff
@@ -39,8 +39,14 @@ load(SlaveNodeName) when is_atom(SlaveNodeName) ->
                         Error -> Error
                     end
             end;
-        _Slave ->
-            ok
+        SlaveNode ->
+            case catch rpc_call(SlaveNode, erlang, round, [5.0]) of
+                5 ->
```

Comment:
Can change this check to some thing more in context, like `erlang:system_info(otp_release).` for example?

Reply:
have changed it to `erlang:montonic_time/0`.